replication-api-impl/src/test/java/org/codice/ditto/replication/api/impl/SyncHelperTest.java

```diff
@@ -64,14 +65,15 @@
   @Mock ReplicationStore destination;
   @Mock ReplicatorConfig config;
   @Mock ReplicationPersistentStore persistentStore;
-  @Mock ReplicatorHistory history;
+  @Mock ReplicatorHistoryManager history;
 
   ReplicationStatus status;
 
   @Before
   public void setUp() throws Exception {
     FilterBuilder builder = new GeotoolsFilterBuilder();
-    status = new ReplicationStatusImpl("test");
+    status = new ReplicationStatusImpl();
```

Comment:
We could have kept the other constructor as an overloaded one. There was no harm in it and did simplify the code.